Python Examples/TestMonitor/DeleteResults/delete_results.py

```diff
@@ -0,0 +1,81 @@
+import uuid
```

Comment:
Please add documentation similar to what we had in create results and steps example

Reply:
Added the documentation